lib/shelley/bench/Latency.hs

```diff
@@ -362,3 +401,19 @@ benchWithShelleyServer tracers action = do
                 block0
                 (gp, vData)
                 (act gp)
+
+    poolConfigsFromEnv = lookupEnv "NO_POOLS" >>= \case
+        Nothing -> pure testPoolConfigs
+        Just "" -> pure testPoolConfigs
+        Just _ -> pure []
+
+    testPoolConfigs =
+        [ -- This pool should never retire:
+          PoolConfig {retirementEpoch = Nothing}
+        -- This pool should retire almost immediately:
+        , PoolConfig {retirementEpoch = Just 3}
+        -- This pool should retire, but not within the duration of a test run:
+        , PoolConfig {retirementEpoch = Just 100_000}
+        -- This pool should retire, but not within the duration of a test run:
+        , PoolConfig {retirementEpoch = Just 1_000_000}
+        ]
```

Comment:
I don't think we need to have stake pools at all in the API latency benchmarks, do we?

Reply:
indeed, fixed in https://github.com/input-output-hk/cardano-wallet/pull/2046/commits/e527debc14af47fcb558e432b16b443388364d68